polaris.shopify.com/content/tools/polaris-cli.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Polaris CLI
+description: Commands for building Shopify Apps with Polaris.
+icon: CodeMajor
+order: 5
+---
+
+[<img src="https://img.shields.io/npm/v/@shopify/polaris-cli.svg?labelColor=f9f9f9&color=dcf5f0" alt="npm version" style="width: 95px" />](https://www.npmjs.com/package/@shopify/polaris-cli)
+
+## Commands
+
+This reference lists the [Shopify CLI][] commands for Polaris.
+
+### migrate
+
+Codemod transformations to help upgrade your Polaris codebase.
```

Comment:
^Note this Shopify CLI install link is under Themes instead of Apps in shopify.dev because there is not installation instruction page under the Apps section 😑

Reply:
Great point and suggested change! 👍 

Only thing I would add is linking to the CLI apps Getting Started instead of the install section in Themes. In the Shopify CLI for apps [Getting Started section](https://shopify.dev/apps/tools/cli#getting-started), it recommends not installing it globally. By using `npx` or `yarn` or `pnpm` directly, devs will always use the latest minor or patch version.